docs/skills/md-to-nextjs-migration.md

```
@@ -0,0 +1,237 @@
+
+```typescript
+import {
+    // 既存のインポート...
+    NEW_ITEMS,
+} from './constants';
+```
+
+### Step 4: Section 関数に JSX カードを追加
+
+カードの構造:
+```tsx
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)